tools/privacyidea-migrate-linotp.py

```diff
@@ -435,6 +438,8 @@ def insert_chunks(conn, table, values, chunk_size=100000):
             if config_obj.MIGRATE.get("tokeninfo") and ti:
                 # Add tokeninfo for this token
                 for (k, v) in ti.items():
+                    if k in config_obj.MIGRATE.get("tokeninfo_unicode_keys", []):
+                        v = unicode(v, erros="ignore")
```

Comment:
Assuming that the value is a unicode string (We read from the table as unicode) we can use
```v = v.encode('ascii', 'ignore').decode('utf8')```
This works in Python 2 and Python 3.

Reply:
Will do

Reply:
done in 97f4c86